blog-service/2024-08-21-search.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Behaviour Change for `count` Operator (Search)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - count
+  - search-operator
+  - log-search
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're happy to announce a change in the behavior of the `count` operator. Previously, the `count` operator was supported by the `Int` data type, which could result in a negative count value when exceeding the maximum integer value. To address this, we are changing the data type of the `count` operator from `Int` to `Long` data type to support the handling of large numeric values. 
```

Comment:
a change to the behaviour of the `count` operator to return `Long` values.  Previously, the `count` operator returned `Int` data type which would result in a negative value when exceeding the maximum integer value. With this change, we are changing  the data type of the `count` operator result from `Int` to `Long` data type.